src/networking/index.js

```diff
@@ -32,32 +30,32 @@ export default class {
   init(config: Config) {
     this._config = config;
 
-    this._maxSubDomain = 20;
+    this._maxSubDomain = 10;
```

Comment:
Is this intentional to limit the number of subdomains to 10 even for our own domain?

Reply:
I thought we would continue to allow the 1 - 20 in our ps.pndsn.com domains.